mobile-widgets/qml/DivePlannerSetup.qml

```diff
@@ -158,12 +158,16 @@ Kirigami.ScrollablePage {
 				}
 				TemplateComboBox {
 					editable: false
+					currentIndex: Planner.dive_mode === Enums.OC ? 0 :
+								  Planner.dive_mode === Enums.CCR ? 1 : 2
```

Comment:
so QML cannot assign an enum to current index and vice versa?
That makes me think that `dive_mode` being an enum is NOT an advantage here. It if was an int we could just say `currentIndex: Planner.dive_mode` correct? Likewise the other way around below.

Reply:
yes QML allows currentIndex = Enums.OC

I did it the more complicated way, because I did not want to UI code to impose a specific order in the enum (which you can argue is being too careful)

if Enums.OC changes value, the text in the combobox needs to be chsnged as well.

I will change to use currentIndex = Enum.* I think this happens in a couple of places.

Reply:
we have many instances where we have string arrays and corresponding enums that need to stay in sync. All this requires, I think, is a comment that says "hey, this needs to stay in sync with that".